File: Java/프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.java
class Solution {
    public int[] solution(int[] array) {
        int max=0;
        int maxindex=0;
        for(int i=0; i<array.length;i++){
            if(array[i]>max){
                max=array[i];
                maxindex=i;
            }
        }
        int[] answer = {max,maxindex};
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.js
function solution(array) {
    var answer = new Array(2);
    var max = 0;
    var maxindex = 0;
    for(var i=0; i<array.length; i++){
        if(array[i]>max) {
            max=array[i];
            maxindex=i;
        }
    }
    answer[0]=max;
    answer[1]=maxindex;
    return answer;
}